docs/reporting-standards.md

```diff
@@ -1,38 +1,50 @@
+ use screenshots and examples liberally so that the norms described here
+ are accessible in both a textual and graphical form.
 
+# General rules
+
+* In line with InaSAFE's human interface guidelines, all titles should capitalise
+  only the first letter of the first word in labels and titles.
+* Titles, body text, table text, table text row heading and table text column
+  headings for text content left aligned.
+* Numbers and table text column headings for number content right aligned.
 
 # Maps
 
 ## Map title
 
 ### Description:
 
-The map title appears at the top of each map product and describes the map content. Using this table as a reference:
-![image](https://cloud.githubusercontent.com/assets/13621886/16513431/eab2319e-3f8e-11e6-9249-eb265f95ac43.png)
+The map title appears at the top of each map product and describes the map
+ content. Evey map should have a title. Using the below table as a reference,
+ the title should use the following syntax:
+
+ <**exposure type**> <**affected by**> <**hazard type**> <**event/hazard**>
 
-We propose the title should use the following syntax
-<**exposure**> <**affected by/in**> <**type of hazard**> <**event/hazard**>
+![image](https://cloud.githubusercontent.com/assets/16660099/17200287/154d7bfa-54ae-11e6-8d7f-ff7017a43d6c.png)
 
-* In the table, we see that **exposure** contains people (population), buildings (building 
-  and infrastructure), roads, and land cover. It is people, property, systems, or other elements 
-  present in hazard zones that are thereby subject to potential losses (UNISDR's terminology).
-* **Affected by/in**, is the influence or effect from external factor, i.e. affected by a disaster.
-* **The type of hazard**, it is the type of hazards that supports by InaSAFE to analyse with.
-* **Event or hazard**, this is a different scenario, in a single scenario we use **event** and 
-  when it is multiple scenarios we use **hazard**.
-* We need to use specific “not translated” words for the Indonesian version
-* Capitalise the first letter of the title only
+**Exposure type** identifies which exposure element present in the analysis
+ area, that is subject to potential losses, is being focused on in this report.
+ Examples of exposure elements include: people (population), buildings
+ (building and infrastructure), roads, and land cover. **Affected by/in** is the
+ influence or effect from an external factor, i.e. affected by a hazard. We
+ use the word 'affected' as the default describer for all hazards. Words such
+ as inundated, flooded, impacted should not be used. Please note that for
+ Indonesian reports we use the word 'terdampak' even though it is not directly
+ translated from 'affected'. **Hazard type** is the type of hazard that has
+ been chosen for analysis out of the hazards that are supported by InaSAFE
+ for analysis. **Event/hazard** shows whether the analysis is from a single
+ scenario (**event**) or multiple scenarios (**hazard**).
```

Comment:
it might be good to use the same terminology as in the key words - ie single event (event), multiple event (hazard)